android_release/11.0.yml

```diff
@@ -0,0 +1,126 @@
+
+  - note: Keyboard navigation bugs resolved
+    tag: fixed
+
+  - note: Star and attachment icon colors corrected
+    short_note: Star and attachment icon colors corrected
+    tag: fixed
+
+  - note: Paste of recipient lists into compose field now correctly parsed
+    tag: fixed
+
+  - note: Back button now returns to Drafts folder when editing a draft
+    tag: fixed
+
+  - note: Unified Inbox now appears as expected
+    tag: fixed
+
+  - note: Dark and light mode switching issues resolved
+    tag: fixed
+
+  - note: Scroll indicator removed following review feedback
+    tag: fixed
+
+  - note: System navigation bar did not correctly blend with message content
+    short_note: System navigation bar did not correctly blend with message content
+    tag: fixed
+
+  - note: '"New Mail Notifications" setting did not persist'
+    short_note: '"New Mail Notifications" setting did not persist'
+    tag: fixed
+
+  - note: Trailing whitespace in recipient fields caused "Incomplete input" error
+    short_note: Trailing whitespace in recipient fields caused "Incomplete input" error
+    tag: fixed
+
+  - note: Navigation drawer "Show Unified inbox" setting did not work
+    short_note: Navigation drawer "Show Unified inbox" setting did not work
+    tag: fixed
+
+  - note: Application could crash after deleting accounts and continuing to use the app
+    short_note: Application could crash after deleting accounts and continuing to use the app
+    tag: fixed
+
+  - note: Sharing multiple selected images from gallery only selected last image
+    short_note: Sharing multiple selected images from gallery only selected last image
+    tag: fixed
+
+  - note: Account drawer could crash when folder was selected
+    short_note: Account drawer could crash when folder was selected
+    tag: fixed
+
+  - note: Application could crash when fetching emails
+    short_note: Application could crash when fetching emails
+    tag: fixed
+
+  - note: Don't cover download complete message with navigation bar
+    short_note: Don't cover download complete message with navigation bar
+    tag: fixed
+
+  - note: Avoid a crash related to the drawer when device orientation changes
+    short_note: Avoid a crash related to the drawer when device orientation changes
+    tag: fixed
+
+  - note: TalkBack did not clearly announce contact picture tap action
+    short_note: TalkBack did not clearly announce contact picture tap action
+    tag: fixed
+
+  - note: IMAP push could fail after losing and regaining network connection
+    short_note: IMAP push could fail after losing and regaining network connection
+    tag: fixed
+
+  - note: Fastmail OAuth expired quickly, causing frequent login prompts
+    short_note: Fastmail OAuth expired quickly, causing frequent login prompts
+    tag: fixed
+
+  - note: Some providers blocked SMTP connections when HELO hostname was 127.0.0.1
+    short_note: Some providers blocked SMTP connections when HELO hostname was 127.0.0.1
```

Comment:
I think this is where the release notes format doesn't really work out. In this case it might make more sense to talk about what we do now rather than what we didn't do. I'd elevate this to a "new" item also.

Reply:
Improved compatibility with some ISPs by using ehlo.thunderbird.net as the EHLO identifier.

Reply:
I agree. Updated and moved to new.